freqtrade/worker.py

```diff
@@ -131,8 +131,7 @@ def _throttle(self, func: Callable[..., Any], throttle_secs: float, *args, **kwa
         return result
 
     def _process_stopped(self) -> None:
-        # Maybe do here something in the future...
-        pass
+        self.freqtrade.process_stopped()
```

Comment:
This will call `cancel_all_open_orders()`, I do not think it's the intended behavior...
This can be fixed/improved either by a flag here or in the Freqtradebot class, or better by introducing the new STOPPING state, which will transit the state from RUNNING to STOPPED. 
(@xmatthias not in this PR, what do you think?)

Reply:
that's a good point - though i'm not sure when / if this would cause problems - once the orders are closed, the loop won't return anything.

On the other hand, assuming i start the bot in stopped state (this sounds contradicting, but can be done by setting `initial_state` to "stopped" in the config) - it would start, sell all trades without ever starting (which may or may not be what i want).

Reply:
Okay, not in this PR (I'll introduce the STOPPING state later)